### ISSUE TITLE
Fractured Fantasy

### ISSUE THREAD

**Issue by millerj-tc** [ACG8/Lucid#260]
![image](https://user-images.githubusercontent.com/3642851/44970123-c2130300-af15-11e8-8e6a-cd9a1beeb58c.png)

**Comment by millerj-tc** [comment]
![image](https://user-images.githubusercontent.com/3642851/44970190-0c947f80-af16-11e8-8350-06fe57a256bd.png)

![image](https://user-images.githubusercontent.com/3642851/44970192-10280680-af16-11e8-94ed-5dbc9240b84b.png)

**Comment by ANRTheBigBoy** [comment]
These are a little on the weak side. If these passives are going to be below curve the bursts are gonna have to be better.

**Comment by ACG8** [comment]
Yeah. Prepared for Rain in particular compares unfavorably with Shelter From the Dark. Definitely buff burst effects.

**Comment by ANRTheBigBoy** [comment]
What if I gain a will from someone else's Beyond the Herd? Does that count?

**Comment by millerj-tc** [comment]
Ah, good catch.
![image](https://user-images.githubusercontent.com/3642851/45039257-e0b5ee80-b028-11e8-8f1f-74679a8fe9ca.png)
![image](https://user-images.githubusercontent.com/3642851/45039272-eb708380-b028-11e8-86cd-2b9798ea71dc.png)
![image](https://user-images.githubusercontent.com/3642851/45039288-f3c8be80-b028-11e8-8f62-142604e4d357.png)
